tests/README.md

```diff
@@ -144,5 +153,41 @@ Bugzilla is about `sshd_config` being not altered correctly. Thus:
 Now, you can perform validation check with command
 
 ```
-./test_suite.py rule --hypervisor qemu:///system --domain ssg-test-suite-centos --datastream ./ssg-centos7-ds.xml rule_sshd_disable_kerb_auth
+./test_suite-vm.py rule --hypervisor qemu:///system --domain ssg-test-suite-centos --datastream ./ssg-centos7-ds.xml rule_sshd_disable_kerb_auth
+```
+
+## Docker backend
+
+You can also use the docker-based for running tests, just use the `test_suite-docker.py` script.
+You need to provide `--base-image <image name>` option on the command-line.
+To obtain the base image, you can use the `Dockerfile` in this directory to build it.
+Therefore, you can build the image and run the test suite like this:
+
+```
+docker build --build-arg CLIENT_PUBLIC_KEY="ssh-rsa AAAAB3NzaC1y...rJSs4BL me@localhost" -t ssg_test_suite .
+./test_suite-docker.py rule --base-image ssg_test_suite --datastream ./ssg-centos7-ds.xml rule_sshd_disable_kerb_auth
```

Comment:
I would restructure it in the following way:

To obtain the base image, you can use the `Dockerfile` in this directory to build it. Build the image using this command:
docker build --build-arg CLIENT_PUBLIC_KEY="ssh-rsa AAAAB3NzaC1y...rJSs4BL me@localhost" -t ssg_test_suite .

Run the test suite using this command:
./test_suite-docker.py rule --base-image ssg_test_suite --datastream ./ssg-centos7-ds.xml rule_sshd_disable_kerb_auth

Reply:
Good idea, thanks!